fla/ops/common/cp_chunk_delta_h.py

```
@@ -0,0 +1,832 @@
+    return x, context.cu_seqlens_conv1d
+
+def post_process_for_conv1d(x, seqlen):
+    context = get_gdn_cp_context()
+    if context.group is None:
+        return x
+    dim = x.dim()
+    if dim == 3:
+        x = x.squeeze(0)
+    x = x[-seqlen:]
+    if dim == 3:
+       x = x.unsqueeze(0)
```

Flake8 (7.3.0)

[error] indentation is not a multiple of 4

(E111)